Makefile

```diff
@@ -9,7 +9,7 @@ format: ## Run the formatter
 TESTS ?= tests
 .PHONY: test
 test: ## Run the test suite
-	@pipenv run python -m pytest -v --cov sdclientapi --cov-report html --cov-report term-missing $(TESTS)
+	@python -m pytest -v --cov sdclientapi --cov-report html --cov-report term-missing $(TESTS)
```

Comment:
why remove pipenv run?

Reply:
This matches the behavior of the `Makefile` in the client repo and I think overall this makes more sense since I would assume one would be in a `pipenv shell` for most dev work anyway.

Reply:
Also when we add other targets that need the `pipenv`, we only end creating a shell once instead of `N` times.

Reply:
ok cool, i don't have strong feelings on this. at some point we should add basic info about setting up a dev env and running tests to the readme in this repo (not requesting in this PR because it is not present on master and not strictly related to the changes here)